packages/components/src/components/shared/FormText.test.tsx

```diff
@@ -0,0 +1,157 @@
+import 'react-native'
+import { render, screen } from '@testing-library/react-native'
+import React from 'react'
+
+import { ComponentWrapper } from '../../wrapper'
+import { Description, Error, Header, Hint, Label } from './FormText'
+
+const mockedColorScheme = jest.fn()
+
+jest.mock('react-native/Libraries/Utilities/useColorScheme', () => {
+  return {
+    default: mockedColorScheme,
+  }
+})
+
+describe('Form Text', () => {
```

Comment:
Seems like at least one set (potentially all of them?) should validate the non-object, string only scenario and that the a11yLabel is the same as the text.

Reply:
Potentially also another should validate the object, but no special a11yLabel--to hedge against future scenarios when there's more than just text/a11yLabel on the object.

Reply:
Added non-object tests for each component and object without a11yLabel tests